static/js/constants_test.js

```diff
@@ -0,0 +1,40 @@
+import _ from 'lodash';
+import { assert } from 'chai';
+
+import { DASHBOARD_RESPONSE } from './constants';
+
+describe('constants', () => {
+  it("doesn't duplicate any id numbers within the same type of information", () => {
+    let programIds : Set<number> = new Set();
+    let courseIds : Set<number> = new Set();
+    let runIds : Set<number> = new Set();
```

Comment:
Might be good to turn it on eventually. Since we're not defining functions or variables for outside use in these files the type checking should hopefully happen without additional annotation needed